### ISSUE TITLE
ruTorrent v3.6 svn $Rev: 2404 $ can't move torrent data files

### ISSUE THREAD

**Issue by ghost** [SynoCommunity/spksrc#1077]
Using the "Save to…" function when right clicking a torrent to move it to another folder is broken.
##

**Comment by GuSoCastillo** [comment]
I recently installed the packe via the package central and got the same problem with the save to... Used to move the files whitout stop seeding. Can anyone explainme where I can download the latest spk for x86?

**Comment by BadFlo** [comment]
Hi GuSoCastillo,

Best way now is to use docker so that you can install any version from any
app store on any system and fiddle with it all you want.

Or you can install last version of rutorrent from SynoCommunity and replace
/web/rutorrent with latest available.

**Comment by GuSoCastillo** [comment]
Thanks badflo I got some questions where I can find docker and more information about it, and where I can download the latest version of rutorrent

**Comment by BadFlo** [comment]
Docker is available on the standard Synology store, and you will be able to
download an all in one rtorrent/rutorrent Docker in it.

For more info about Docker, check Synology website and Docker documentation
online.

;)

**Comment by GoodOmens83** [comment]
The version in the package center is outdated for the qoriq.

I complied the latest from source and uploaded to my Synology via the manual install.  Still didn't work.

I then noticed I had user-settings.ini /etc/php/conf.d.  In that file was a line also setting open_basedir which conflicts with com.synocommunity.packages.rutorrent.ini created as part of the install.

Deleting the open_basedir line in /etc/php/conf.d/user-settings.ini fixed the issue for me.  Maybe going forward have the rutorrent installer comment out that line and then uncomment on uninstall?
